packages/twenty-front/src/modules/object-record/record-index/options/components/RecordIndexOptionsDropdownContent.tsx

```diff
@@ -143,10 +176,28 @@ export const RecordIndexOptionsDropdownContent = ({
     navigationMemorizedUrlState,
   );
 
+  const viewGroupMenuItem =
```

Comment:
Prefer defining a boolean variable like `shouldShowSubMenuItem` and put the condition in the return part otherwise we handle JSX outside of the return part and that makes it harder to keep a coherent routine of reading JSX only in the return part.